exchanges/bybit/bybit_websocket.go

```diff
@@ -475,7 +475,7 @@ func (by *Bybit) wsHandleData(respRaw []byte) error {
 						}
 					}
 
-					p, err := by.extractCurrencyPair(data[j].Symbol, asset.Spot)
+					p, err := by.MatchSymbolWithAvailablePairs(data[j].Symbol, asset.Spot, false)
```

Comment:
This is changing behaviour in all instances of this file. `extractCurrencyPair` was for enabled currencies and so this will need to change